src/sentry/integrations/github/integration.py

```diff
@@ -106,6 +106,20 @@
         """,
         IntegrationFeatures.ISSUE_BASIC,
     ),
+    FeatureDescription(
+        """
+        Automatically synchronize assignees to and from GitHub. Don't get confused
+        who's fixing what, let us handle ensuring your issues and tickets match up
+        to your Sentry and GitHub assignees.
+        """,
+        IntegrationFeatures.ISSUE_SYNC,
+    ),
+    FeatureDescription(
+        """
+        Synchronize Comments on Sentry Issues directly to the linked GitHub issue.
```

Comment:
nit: I know the other ones have capital C but it looks funky

```suggestion
        Synchronize comments on Sentry Issues directly to the linked GitHub issue.
```